website/docs/Use-Cases/agent_chat.md

```diff
@@ -113,8 +113,10 @@ def exchange_rate(base_currency: CurrencySymbol, quote_currency: CurrencySymbol)
     else:
         raise ValueError(f"Unknown currencies {base_currency}, {quote_currency}")
 
-# NOTE: for Azure OpenAI, API version up to 2023-10-01-preview
-# uses the old function call style. So you need to set `api_style="function"`:
+# NOTE: for Azure OpenAI, please use API version 2023-12-01-preview or later as
+# support for earlier versions will be deprecated.
+# For API versions 2023-10-01-preview or earlier you may
```

Comment:
Update this part

Reply:
I think 2023-10-01-preview exists as listed in the API spec repo. https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview